ts/nni_manager/main.ts

```diff
@@ -66,6 +67,10 @@ async function initContainer(foreground: boolean, platformMode: string, logFileN
         Container.bind(TrainingService)
             .to(RouterTrainingService)
             .scope(Scope.Singleton);
+    } else if (platformMode === 'heterogeneous') {
```

Comment:
It looks right time to simplify the code. For example, create a router list, and put supported mode in it. If some string in the list, it should be processed in RouterTrainingService.

Reply:
fixed